src/main/java/org/cloudbees/literate/jenkins/publishers/Agent.java

```diff
@@ -76,7 +76,7 @@ protected Agent(Class<P> clazz) {
      */
     @CheckForNull
     public abstract P getPublisher(@NonNull BuildListener listener, @NonNull FilePath configurationFile)
-            throws IOException;
+            throws IOException, InterruptedException;
```

Comment:
Yes, [as of newer Jenkins versions](https://github.com/jenkinsci/jenkins/commit/c026005#diff-d0a9bd4f4582762fd1d1f31a88d4753cL1666).